conf/cassandra.yaml

```diff
@@ -359,7 +359,7 @@ prepared_statements_cache_size_mb:
 # NOTE: if you reduce the size, you may not get you hottest keys loaded on startup.
 #
 # Default value is empty to make it "auto" (min(5% of Heap (in MB), 100MB)). Set to 0 to disable key cache.
-key_cache_size_in_mb:
+key_cache_size:
```

Comment:
why do we leave valueless settings uncommented?

Reply:
From the comments: # Default value is empty to make it "auto" (min(5% of Heap (in MB), 100MB)). Set to 0 to disable key cache.

Reply:
Sure, but is an empty value the same as a commented out setting? (look at `repair_session_space` which is commented out but also has a calculated default value…)

Reply:
Fair point, I will make them consistent. I just verified that by default in config it is null so it won't change any behavior. Thank you for pointing it

Reply:
That made me think and I found that the same applies to `index_summary_capacity`